app/schemas/com.github.premnirmal.ticker.repo.QuotesDB/6.json

```diff
@@ -2,11 +2,11 @@
   "formatVersion": 1,
   "database": {
     "version": 6,
-    "identityHash": "a9411595d733a02f3a0d29b7aba56959",
```

Comment:
Prefer this to be untouched and a new version 7 schema be added to this folder

Reply:
Updated. I still need some help with making sure the 6->7 migration runs :)